docs/tutorials/mastering.md

```diff
@@ -0,0 +1,636 @@
+# Mastering
+
+In this tutorial, we will deploy and run the SD-Core 5G core network following Control and User Plane Separation (CUPS) principles. The radio and cell phone simulator will also be deployed on an isolated cluster.
+
+```{image} ../images/sdcore_networking.png
+:alt: Network diagram
+:align: center
+```
+
+To complete this tutorial, you will need four machines (virtual or physical), with access to the networks as follows.
+
+| Machine | CPUs | RAM | Disk | Networks |
+| ------- | ---- | --- | ---- | -------- |
+| Control Plane Kubernetes Cluster     | 4 | 8g | 40g | `management` |
+| User Plane Kubernetes Cluster        | 4 | 4g | 20g | `management`, `access`, `core` |
+| Juju Controller + Kubernetes Cluster | 2 | 8g | 40g | `management` |
+| gNB Simulator Kubernetes  Cluster    | 4 | 4g | 20g | `management`, `ran` |
+
+
+## Sample Values
+
+For the purpose of this guide, the following values will be used.
+
+### Networks
+
+| Name | Subnet | Gateway IP |
+| ---- | ------ | ---------- |
+| `management` | 10.201.0.0/24 | 10.201.0.1 |
+| `access`     | 10.202.0.0/24 | 10.202.0.1 |
+| `core`       | 10.203.0.0/24 | 10.203.0.1 |
+| `ran`        | 10.204.0.0/24 | 10.204.0.1 |
+
+### IP Addresses
+
+The following IP addresses are used in this tutorial and must be present in the DNS Server that all hosts are using.
+
+| Name | IP Address | Purpose |
+| ---- | ---------- | ------- |
+| `juju-controller.mgmt` | 10.201.0.10 | Management address for Juju machine |
+| `control-plane.mgmt` | 10.201.0.11 | Management address for control plane cluster machine |
+| `user-plane.mgmt` | 10.201.0.12 | Management address for user plane cluster machine |
+| `gnbsim.mgmt` | 10.201.0.13 | Management address for the gNB Simulator cluster machine |
+| `api.juju-controller.mgmt` | 10.201.0.50 | Juju controller address |
```

Comment:
At this stage, juju controller address is not available, so dns record could not be set yet.